cobalt/site/docs/development/setup-linux.md

```diff
@@ -199,3 +201,222 @@ to debug application performance, but also a great way to debug issues and
 better understand Cobalt's execution flow in general.
 
 Simply build and run one of these configs and observe the terminal output.
+
+# Running Cobalt in evergreen mode
+
+As [Evergreen support](starboard/doc/evergreen/cobalt_evergreen_overview.md) is
+mandatory since 2024 requirement, Cobalt should be executed in Evergreen mode.
+
+## Download and configure the official Google-built Cobalt binaries from GitHub
+1. Create output directory for evergreen
+
+    - Create the directory with arguments that meet your need
+      - Build type, ex: gold, qa
+      - Starboard API version, ex: 15, 16
+    - An example to create directory with build_type=qa and sb_api_version=16
+      ```sh
+      # go to cobalt root folder
+      export COBALT_SRC=${PWD}
+      export PYTHONPATH=${PWD}:${PYTHONPATH}
+
+      # setup evergreen build arguments
+      export EG_PLATFORM=evergreen-x64
+      export EG_BUILD_TYPE=qa
+      export SB_API_VER=16
+      export EVERGREEN_DIR=out/${EG_PLATFORM}_${EG_BUILD_TYPE}
+
+      gn gen $EVERGREEN_DIR --args="target_platform=\"$EG_PLATFORM\" use_asan=false build_type=\"$EG_BUILD_TYPE\" sb_api_version=$SB_API_VER"
+      ```
+
+1. Select Google-prebuilt Cobalt binaries from [GitHub](https://github.com/youtube/cobalt/releases)
+
+    - Choose the correct evergreen version based on the target device specification
+
+      Please note that the selected prebuilt binary must meet the settings used to create evergreen directory in previous step.
+      - Cobalt version you checked out, ex: 25.lts.1+
```

Comment:
nit: consider using `25.lts.1` here so that the version can be matched exactly with the version of the Cobalt core binary.

Reply:
This is changed to 25.lts.1.

Reply:
Thanks!